clang/test/CodeGen/aie/aie-abi-cpp.cpp

```diff
@@ -1,4 +1,4 @@
-// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --function-signature --return-type --skip-function-body
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --function-signature
```

Comment:
Are `--return-type --skip-function-body` not needed anymore? Or did you manually update the test? If so, please remove the header like altogether.

Reply:
I had to update these tests manually, `--return-type --skip-function-body` doesn't exist in aie-public.
I'll drop these headers for now